### DISCUSSION TITLE
Error message shows "SW radiation is negative!!" while using generic metforcing. However there are no negative or Nan values in the downward shortwave radiation.

### DISCUSSION THREAD

**Discussion by PriDeHydro** [NASA-LIS/LISF#1539] [Q&A] [answered]
Hi,
I am trying to run NoahMP4.0.1 with Hymap2 at 0.1 degrees and a 1-hour time-step over the portion in southern India using forecasted forcing.
A forecasted forcing at 4km spatial resolution and daily time-step made to resemble LDT Generated forcing with the following attributes
![ncm_attr](https://github.com/NASA-LIS/LISF/assets/136304021/fda692dc-c383-4699-a1f9-7883ac831272)

MERRA2 is used as a global forcing that provides forcing in the other part of India where the forecasted forcing is not available. 

The lis config is configured to use MERRA2 and ldt-generated forcing.

However, while running the LIS over the complete domain, the run gets initiated and LIS shows an error message as "SW radiation is negative!!" 

![ncm_err](https://github.com/NASA-LIS/LISF/assets/136304021/f309fcc3-5e93-47f4-8c80-8d2f552db370)

The PBS error log shows
![ncm_pbs_err_lg](https://github.com/NASA-LIS/LISF/assets/136304021/9d998b29-8f68-4d93-927d-5921fc050a0c)

**I have checked the generic metforcing for any negative or NaN values but everything seems in order. Also, the runs are successful if I used SW radiation from MERRA2 in place of the generic met forcing but fail when generic forcing is used.**

I am attaching generic metforcing used for the simulation, lisconfig, and lislog outputs. I would appreciate any help regarding this @dmocko , @karsenau .

Thanks.
[files_ref.zip](https://github.com/NASA-LIS/LISF/files/15321072/files_ref.zip)

**Comment by karsenau** [comment, marked as answer]
Hi @PriDeHydro:

I was wondering if you could help describe a bit more about the "forecasted forcing" that you created for your region.  I see that you mentioned "daily time-step".  Is that the time-step interval that your created forcing is at?  If so, NoahMP could have issues, especially in the radiation fields and energy balance to such a coarse temporal resolution in the forcing.

Also, make sure not to use -9999 for the SWdown or other fields, other than fill regions.  You LDT log file indicates that it is interpreting an undefined value of -9999 as an actual value, instead of 0 as the minimum. You might want to check how some of your forcing fields (e.g., swdown and precipitation) are represented for 0-values.
